src/node_i18n.cc

```diff
@@ -461,6 +461,9 @@ int32_t ToUnicode(MaybeStackBuffer<char>* buf,
                                   &status);
   }
 
+  if (info.errors != 0)
+    info.errors &= ~UIDNA_ERROR_EMPTY_LABEL;
```

Comment:
I'd leave off the `if` as the `&` will do nothing if the error is not present, and I'd add a reference to https://url.spec.whatwg.org/#concept-domain-to-ascii (especially the *VerifyDnsLength* bit) for why we are ignoring this error.